### PR TITLE
Update docker-compose.yml scan-runner tag

### PR DESCRIPTION
**Pull request by leonid-zats**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -126,7 +126,7 @@ services:
       - sca-agent
   ########################################################################
   scan-runner:
-    image: cxsca/scan-runner:2021.02.17.1416-099aced
+    image: cxscaci/scan-runner:2021.03.02.1902-10c3870
```
Comment (maor-rosenfeld) [inline docker-compose.yml:129]: no

Comment (leonid-zats) [reply docker-compose.yml:129]: what do you mean? lumops script did this

Comment (maor-rosenfeld) [reply docker-compose.yml:129]: and lev will fix :)

Comment (leonid-zats) [reply docker-compose.yml:129]: whats the manual fix for now, cxsca?

Comment (maor-rosenfeld) [reply docker-compose.yml:129]: yup, but you need to push the image to that repository.

Comment (leonid-zats) [reply docker-compose.yml:129]: I'll wait for the fix to be sure

**Review by maor-rosenfeld — APPROVED** [review]

**Pull request merged**